python/orca/src/bigdl/orca/ray/ray_on_spark_context.py

```diff
@@ -546,19 +547,17 @@ def init(self, driver_cores=0):
                 if self.env:
                     os.environ.update(self.env)
                 import ray
-                kwargs = {}
-                if self.extra_params is not None:
-                    for k, v in self.extra_params.items():
-                        kw = k.replace("-", "_")
-                        kwargs[kw] = v
+                kwargs = self._update_extra_params(self.extra_params)
                 init_params = dict(
                     num_cpus=self.ray_node_cpu_cores,
-                    _redis_password=self.redis_password,
                     object_store_memory=self.object_store_memory,
                     include_dashboard=self.include_webui,
                     dashboard_host="0.0.0.0",
                     _system_config=self.system_config
                 )
+                if "redis_password" in kwargs:
+                    kwargs["_redis_password"] = kwargs["redis_password"]
+                    kwargs.pop("redis_password")
```

Comment:
Can put this part into `update_extra_params`?

Reply:
sure, updated!